doc/ota_updates/ota_updates.md

```diff
@@ -79,7 +79,13 @@ void onError(OTA_CALLBACK_ERROR (fn));
 
 ### Basic Requirements
 
-- Flash chip size is 2x the size of the sketch.
+Flash chip size needs a size thats is able to hold the old sketch (currently running) and the new sketch (OTA) at the same time.
+keep in mind that the File system and EEPROM for example needs space too (one time) see [flash layout](https://github.com/esp8266/Arduino/blob/master/doc/filesystem.md#flash-layout).
```

Comment:
This should be a relative link, (i.e. `../filesystem.md#flash-layout`), otherwise it will be broken in HTML docs.